images/Dockerfile.agent

```diff
@@ -20,6 +20,6 @@ FROM alpine:3.16
 WORKDIR /
 COPY --from=builder /workspace/bin/infraagent /infraagent
 COPY --from=builder /workspace/bin/felix-api-proxy /felix-api-proxy
-RUN apk --no-cache -U add iproute2-rdma~=5.17
+# RUN apk --no-cache -U add iproute2-rdma~=5.17
```

Comment:
Why is this commented out? We need it for CDQ

Reply:
Is it needed for basic functionality or for troubleshooting?

Reply:
Reverted it back.